wandb/integration/keras/keras.py

```diff
@@ -958,5 +971,31 @@ def _save_model(self, epoch):
         # Was getting `RuntimeError: Unable to create link` in TF 1.13.1
         # also saw `TypeError: can't pickle _thread.RLock objects`
         except (ImportError, RuntimeError, TypeError) as e:
-            wandb.termerror("Can't save model, h5py returned error: %s" % e)
+            wandb.termerror(
+                "Can't save model in the h5py format. The model will be saved as "
+                "W&B Artifacts in the SavedModel format."
+            )
             self.save_model = False
+            self.save_model_as_artifact = True
+
+    def _save_model_as_artifact(self, epoch):
+        if wandb.run.disabled:
+            return
+        try:
+            # Save the model in the SavedModel format.
+            self.model.save(self.filepath[:-3], overwrite=True, save_format="tf")
+
+            # Log the model as artifact.
+            model_artifact = wandb.Artifact(f"model-{wandb.run.name}", type="model")
+            model_artifact.add_dir(self.filepath[:-3])
+            wandb.run.log_artifact(
+                model_artifact, aliases=["latest", "best", f"epoch_{epoch}"]
```

Comment:
for the aliases, i don't see the benefit of `best` - since it is always paired with the `latest` in this case. Seems a bit redundant. Also, the `epoch_` may be redundant as well (since the version index is the epoch). If these are both true, i'd like to remove the aliases here.

Reply:
You are right, I too don't see a benefit of `best` as an alias. However, I think `epoch_` is still useful. 

The method `_save_model_as_artifact` is called when `if self.current and self.monitor_op(self.current, self.best):` is true. While training a model this condition might not be true every time and the model will be saved as an artifact at those (where it's true) epochs. Thoughts?